msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -587,6 +700,92 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
         assertTrue((accessTokenState as GetAccessTokenError).isNoAccountFound())
     }
 
+    /**
+     * Test sign in, sign out, get access token
+     */
+    @Test
```

Comment:
Is this test for the sign out part. It doesn't feel like a good test for the getAccessToken part.

Reply:
This test is a slight variation of the test `testSignOutGetAccessToken()` that calls the `getAccessToken` with two parameters.